packages/types/src/core/relayer.ts

```diff
@@ -119,4 +123,5 @@ export abstract class IRelayer extends IEvents {
   public abstract restartTransport(relayUrl?: string): Promise<void>;
   public abstract confirmOnlineStateOrThrow(): Promise<void>;
   public abstract handleBatchMessageEvents(messages: RelayerTypes.MessageEvent[]): Promise<void>;
+  public abstract onLinkMessageEvent(messageEvent: RelayerTypes.MessageEvent): Promise<void>;
```

Comment:
Should this be under relayer given the fact that LinkMode is a relay-less connection? Should we "separate concerns"?